test/tests.js

```diff
@@ -52,11 +52,16 @@ window.onload = function(){
 
 		asyncTest( 'thumbnail clicks cause pane to scroll', function() {
 			expect(2);
+			var originalHash = window.location.hash;
 			$(".snapper_pane")[0].scrollLeft = 0;
 			$(".snapper_nav a:eq(2)").trigger( "click" );
 			setTimeout(function(){
 				ok( $(".snapper_pane")[0].scrollLeft !== 0, "scroll changed" );
-				ok( window.location.hash === $(".snapper_nav a:eq(2)").attr( "href" ), "location hash set to active slide" )
+				if( $(".snapper").attr( "data-snapper-deeplinking" ) === "false" ) {
```

Comment:
Uh, I do think this would be better, but the tests are super coupled to only having one example on the test page (if you look at `test/index.html`). So since this is non-default behavior, it’d require another markup example and I’d have to rewrite a bunch of tests?